tests/test_points_allocators.py

```diff
@@ -0,0 +1,159 @@
+import unittest
+
+import torch
+from aepsych.config import Config
+from aepsych.models.inducing_point_allocators import (
+    AutoAllocator,
+    KMeansAllocator,
+    SobolAllocator,
+)
+from aepsych.models.utils import select_inducing_points
+from botorch.models import SingleTaskGP
+from botorch.models.utils.inducing_point_allocators import (
+    GreedyImprovementReduction,
+    GreedyVarianceReduction,
+    InducingPointAllocator,
+)
+from botorch.utils.sampling import draw_sobol_samples
+
+
+class TestInducingPointAllocators(unittest.TestCase):
+    def test_sobol_allocator_from_config(self):
+        config_str = """
+            [common]
+            parnames = [par1]
+
+            [par1]
+            par_type = continuous
+            lower_bound = 0.0
+            upper_bound = 1.0
```

Comment:
Check if bounds are correct if a transformation is applied (easiest to check is `log_scale = True`).

Reply:
It must be tested here, transforms are almost always applied (the Normalize transform), the log_scale is simply the most obvious when something goes wrong.